docs/admin-manual/configuration-macros.rst

```diff
@@ -6155,6 +6160,14 @@ These settings affect the *condor_starter*.
     *condor_shadow* and *condor_startd* daemons. Defaults to 300 (5
     minutes).
 
+:macro-def:`STARTER_INITIAL_UPDATE_INTERVAL[STARTER]`
+    An integer value representing the number of seconds before the
+    first ClassAd update from the *condor_starter* to the *condor_shadow*
+    and *condor_startd*.  Defaults to 2 seconds.  On extremely
+    large systems which frequently launch all startds at the same time,
```

Comment:
Do we mean `launch all startds` here or is it supposed to be starters?

Reply:
Could be either.  I guess starter is more correct.